xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/ExportAction.java

```diff
@@ -104,20 +104,23 @@
          */
         private String description;
 
-        ExportArguments(XWikiContext context) throws XWikiException
+        ExportArguments(XWikiContext context, String format) throws XWikiException
```

Comment:
An alternative would be to pass the default name (that you can overwrite from the request) instead of the format, but it depends if you think the format might be useful in ``ExportArguments`` in the future for something else than determining the default name.

Reply:
Yeah I think the format would be more useful to process the arguments differently in the future. I actually hesitated in creating an enum for the different formats instead of relying on a string... Could be cleaner.